scraper.go

```diff
@@ -76,18 +75,12 @@ func NewScraper(conf *Config) (Scraper, error) {
 func newScheduledCollector(conf *Config, pipelineGroups PipelineGroups) (
```

Comment:
nit: the `PipelineGroups` argument is no longer needed.

Reply:
Indeed!